components/Map/index.js

```diff
@@ -0,0 +1,144 @@
+import { MapContainer, Marker, Popup, TileLayer } from "react-leaflet";
+import styled from "styled-components";
+import "leaflet/dist/leaflet.css";
+import "leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.css";
+import "leaflet-defaulticon-compatibility";
+import LocationMarker from "./LocationMarker";
+import * as L from "leaflet";
+import { useState } from "react";
+import Link from "next/link";
+const StyledMapContainer = styled(MapContainer)`
```

Comment:
nitpicking right here, but an empty line would make it more readable :) 

```suggestion
import Link from "next/link";

const StyledMapContainer = styled(MapContainer)`
```

Reply:
done